src/components/AboutModal/AboutModal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Icon, Modal } from '../../index';
+
+const AboutModal = ({
+  children,
+  className,
+  show,
+  onHide,
+  productTitle,
+  logo,
+  altLogo,
+  trademarkText,
+  ...rest
+}) => {
+  return (
+    <Modal
+      className={className}
+      contentClassName="about-modal-pf"
+      show={show}
+      onHide={onHide}
+      {...rest}
+    >
+      <Modal.Header>
+        <Button
```

Comment:
nit here I am just noticing... using `Button` component is adding the `btn btn-default` class to the close button and making hover the wrong background color for our PF About Modal bg. I had the following workaround before:
```            
<button className="close" onClick={this.close} aria-hidden="true" aria-label="Close">
````
The only way I've found to fix this with RB Button is setting `bsClass=" "` (empty space).

A better alternative might be adding `Modal.CloseButton` and just using `<button>`.